docs/integrations/ember-and-json-api.md

```diff
@@ -12,6 +13,21 @@ Note: This guide assumes that `ember-cli` is used for your ember app.
 The JSON API specification calls for hyphens for multi-word separators. AMS uses underscores.
 To solve this, in Ember, both the adapter and the serializer will need some modifications:
 
+### Server-Side Changes
+
+there are multiple mimetypes for json that should all be parsed similarly, so
+in `config/initializers/mime_types.rb`:
+```ruby
+api_mime_types = %W(
+  application/vnd.api+json
+  text/x-json
+  application/json
+)
+
+Mime::Type.unregister :json
+Mime::Type.register 'application/json', :json, api_mime_types
```

Comment:
We could certainly add that later - maybe when the JSON API implementation becomes more complete. I think that should be separate from this PR though.